libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/SalesforceActivity.java

```diff
@@ -146,6 +151,12 @@ public void authenticatedRestClient(RestClient client) {
 	}
 
     /**
+     * Performs actions on logout complete.
+     */
+    protected void logoutCompleteActions() {
```

Comment:
Default implementation in the `Salesforce` activities is nice, so that typical app `Activity` gets this for free.